### ISSUE TITLE
Progress Steps not changed when updating total

### ISSUE THREAD

**Issue by EvanSchalton** [tqdm/tqdm#883]
- [x] I have marked all applicable categories:
    + [ ] exception-raising bug
    + [x] visual output bug
    + [ ] documentation request (i.e. "X is missing from the documentation." If instead I want to ask "how to use X?" I understand [StackOverflow#tqdm] is more appropriate)
    + [ ] new feature request
- [x] I have visited the [source website], and in particular
  read the [known issues]
- [x] I have searched through the [issue tracker] for duplicates
- [x] I have mentioned version numbers, operating system and
  environment, where applicable:
  ```python
  import tqdm, sys
  print(tqdm.__version__, sys.version, sys.platform)
  ```
```4.36.1 3.7.1 (default, Dec 10 2018, 22:54:23) [MSC v.1915 64 bit (AMD64)] win32```

Love the tool! I have an odd use-case where I don't know the total at the beginning so I update it as it grows. This works really well for the % calculation, but the bar itself is confined to the number of iterations initially set in total.

```
from tqdm import tqdm_notebook as tqdm
prog=tqdm(total=500, dynamic_ncols=True)
prog.total = 1000
prog.update(500) #shows a full progress bar (incorrect) & 50% metric (correct)
```

In case I'm just missing a method, I posted a more complete example on S/O:
https://stackoverflow.com/questions/59798036/tqdm-notebook-increase-bar-width

  [source website]: https://github.com/tqdm/tqdm/
  [known issues]: https://github.com/tqdm/tqdm/#faq-and-known-issues
  [issue tracker]: https://github.com/tqdm/tqdm/issues?q=
  [StackOverflow#tqdm]: https://stackoverflow.com/questions/tagged/tqdm

**Comment by casperdcl** [comment]
Hmm I think perhaps the highly memorable incantation `prog.container.children[0].max = prog.total` might work

**Comment by EvanSchalton** [comment]
@casperdcl if you want some Stack Overflow cred post the answer there too and I'll accept

Thanks for the help, I dug around for a solid hour or two before posting - might be nice to see this as the default behavior in the total setter.

**Comment by casperdcl** [comment]
> might be nice to see this as the default behavior in the total setter

Exactly :)

**Comment by EvanSchalton** [comment]
I tried to create a PR, but it's restricted

Should be able to paste this
```
    @property
    def total(self):
        return self.__total

    @total.setter
    def total(self, total):
        self.__total = total
        self.container.children[0].max = total
```

into `notebook.py` at line 265

**Comment by casperdcl** [comment]
> but it's restricted

What do you mean? You should be able to fork & PR

**Comment by casperdcl** [comment]
actually this is a duplicate of #864 which was fixed by #866 (`tqdm>=4.41.0`). Just use `.reset(total=<new_total>)`
